scipy-1.0/paper.tex

```diff
@@ -258,6 +262,20 @@ \subsubsection*{LowLevelCallable}
 
 \subsection*{Cython bindings for BLAS, LAPACK and special}
 
+When writing algorithms or wrappers in Cython, it is often convenient to call directly into the libraries that SciPy wraps rather than routing the call back through the Python interpreter.
```

Comment:
Regarding "call directly into the libraries that SciPy wraps rather than routing the call back through the Python interpreter", does this mean e.g. importing `scipy.special.cython_special` and calling `scipy.special.cython_special.jve(...)` rather than importing `scipy.special` and calling `scipy.special.jve(...)`?